rules/windows/persistence_user_account_added_to_privileged_group_ad.toml

```diff
@@ -15,7 +15,39 @@ index = ["winlogbeat-*", "logs-windows.*", "logs-system.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "User Added to Privileged Group in Active Directory"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating User Added to Privileged Group in Active Directory
+
+Privileged accounts and groups in Active Directory are those to which powerful rights, privileges, and permissions are
+granted that allow them to perform nearly any action in Active Directory and on domain-joined systems.
+
+Attackers can add users to privileged groups to maintain their level of access if other privileged accounts they do have
+access to get uncovered by the security team. This way, they can continue their operations even after the discovery of
+an abused account.
+
+This rule monitors events related to the addition of a user to one of the privileged groups.
+
+#### Possible investigation steps
+
+- Identify the user account that performed the action and whether it should manage members of this group.
+- Contact the account owner and confirm whether they are aware of this activity.
+- Investigate other alerts associated with the user during the past 48 hours.
+
+### False positive analysis
+
+- This attack abuses a legitimate Active Directory mechanism, so it is important to determine whether the activity is
```

Comment:
This sentence is a big long so I'd break this up into two sentences, for example:
This attack abuses a legitimate Active Directory mechanism. It is important to determine whether the activity is...